cli/src/main/java/com/devonfw/tools/ide/commandlet/CreateCommandlet.java

```diff
@@ -28,6 +36,8 @@ public CreateCommandlet(IdeContext context) {
     super(context);
     this.newProject = add(new StringProperty("", true, "project"));
     this.skipRepositories = add(new FlagProperty("--skip-repositories"));
+    this.codeRepositoryFlag = add(new FlagProperty("--code"));
+    this.codeRepository = add(new StringProperty("", false, "codeRepository"));
```

Comment:
We do not need an extra `codeRepository` since then the user can provide two URLs what will be rather confusing.

Reply:
done

Reply:
Did you forget to push your changes?

Reply:
Not really, its really weird that you are not seeing the latest changes

Reply:
Was a refresh problem. Sorry for confusion. Thanks for the great rework. Resolved.